packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.tsx

```diff
@@ -32,6 +34,13 @@ export class TdsTableBodyRowExpandable {
    *  Take in mind that expandable control is column too */
   @Prop() colSpan: number = null;
 
+  /** Sets default expanded value of row */
+  @Prop() defaultExpanded: boolean;
```

Comment:
Good suggestion, like you said, maybe changing to prop instead:

`@Prop({ mutable: true, reflect: true }) isExpanded: boolean = false;`

Reply:
That's how I started out, but then the component ended up in a confused state since default is false, Therefore when making isExpanded a prop, it starts as false so when you try to collapse it changes to true and remains open. Then you have to press it again to collapse.

I could look into this again if needed, but i'll leave it for another PR in that case.

Reply:
I think it's good to differentiate between a controlled vs an uncontrolled component state. To me `default-expanded` makes sense for a user to be able to rely on the component to "handle" itself after the inital render. But adding a `expanded` prop that is fully controlled by the user and we never update the expanded/collapsed state if this is used also makes sense.